core/main/src/bootstrap/start_fbgateway_step.rs

```diff
@@ -50,6 +50,9 @@ impl FireboltGatewayStep {
         let _ = methods.merge(DeviceRPCProvider::provide_with_alias(state.clone()));
         let _ = methods.merge(WifiRPCProvider::provide_with_alias(state.clone()));
         let _ = methods.merge(LifecycleRippleProvider::provide_with_alias(state.clone()));
+        let _ = methods.merge(LifecycleManagementProvider::provide_with_alias(
```

Comment:
Missing, needed to pull in LifecycleManagement RPCs.

Reply:
I was able to make `lifecyclemanagement.session() `call in gateway mode (no launcher).  @pahearn73 : Is your change applicable only in launcher mode ?

Reply:
Lifecyclemanagement Provider is only used if we don't have an internal launcher please check [this](https://github.com/rdkcentral/Ripple/blob/7ee4a795ce5ce1ca111ff30710616a5f83ee9a5c/core/main/src/bootstrap/start_fbgateway_step.rs#L76) 

If the `firebolt-extn-manifest.json` contains a launcher extension it is in gateway-with-launcher if it doesn't contain the launcher extension by default it will be in gateway mode which needs the lifecyclemanagement provider.